traffic_ops_ort/testing/README.md

```diff
@@ -0,0 +1,102 @@
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Traffic Ops ORT Tests
+
+The ORT tests are used to validate the ORT tools used with a
+release of Traffic Ops.  The tests ensure that the ORT tools
+are able to communicate with Traffic Ops, install required
+packages and generate the correct ATS configuration files on
+a Trafficserver cache.
+
+The first thing you need do is to provide the Traffic Ops and
+Traffic Ops ORT RPM files for the build you wish to test using 
+this framework.  This test environment provides the necessary
+Docker containers used to support and execute the tests when
+given the necessary RPM's.  If you choose to not use the provided
+Docker containers you will need to provide the following resources:
+
+  - A running Traffic Ops with the installed release to be tested
+  - A running Postgres SQL database which is loaded with the proper
+    test data for the Traffic Ops release.
+  - An Apache Traffic Server host that has the installed release of
+    ORT to be tested against the release of Traffic Ops.
+  - A yum server configured to provide the test rpm's herein.
+  - A Traffic vault server.
+
+# Directory layout
+
+  - trafficcontrol/traffic_ops_ort/testing/docker:  has all the 
+    necessary files for running the test Docker containers.
+  - trafficcontrol/traffic_ops_ort/testing/ort-tests:  this directory.
+    contains all the go files used to run the ORT tests.
+
+# Setup.
+
+  1.  Build the Traffic Ops and Traffic Ops ORT RPM's that you wish
+      to test.  See the top level 'build' directory for building 
+      instructions. 
+  2.  Copy the Traffic Ops RPM to docker/trafffic_ops/traffic_ops.rpm 
+      (NOTE:  Use the file name 'traffic_ops.rpm')
+  3.  Copy the Traffic Ops ORT rpm to docker/ort_test/traffic_ops_ort.rpm
+      (NOTE:  Use the file name 'traffic_ops_ort.rpm'
+  4.  Copy the recent copy of riak-2.2.3-1.el7.centos.x86_64.rpm to
+      docker/traffic_vault/riak-2.2.3-1.el7.centos.x86_64.rpm
+  5.  Copy an Apache Trafficserver rpm to 
+      docker/yumserver/test-rpms/trafficserver-$VERSION.$COMMIT-HASH.el7.x86_64.rpm
+      You will need to edit and adjust the trafficserver package value in
+      ort-tests/tc-fixtures.json to match the $VERSION.$COMMIT-HASH used in the name
+      of your RPM.  Search for '8.0.8-19.77cb23a' in the ort-tests/tc-fixtures.json 
+      and change the value to match the RPM version you choose to use.
+      For example, the current value in tc-fixtures.json is '8.0.8-19.77cb23a' and 
+      therefore the rpm file expected in 'docker/yumserver/test-rpms' is
+      'trafficserver-8.0.8-19.77cb23a.el7.x86_64.rpm'.
```

Comment:
The text "8.0.8-19.77cb23a" does not appear in `tc-fixtures.json`

```shell-session
 ocket8888@system76  testing  ➦ 63ba71cf1  grep "8.0.8" ./ort-tests/tc-fixtures.json 
 ✘ ocket8888@system76  testing  ➦ 63ba71cf1  
```

Reply:
@ocket8888 resolved.